src/styles/main.scss

```
@@ -1,3 +1,223 @@
-body {
+* {
   margin: 0;
 }
```

```suggestion
body {
  margin: 0;
}

* {
  box-sizing: border-box;
}
```
use * selector only for box-sizing property. reset margins for tags which need it